packages/app/src/components/Normal/FloatingActionButton/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+
+import { Meta, Story } from "@storybook/react"
+import { ThemeProvider } from "styled-components"
+import { youtubeLiteTheme } from "src/styles/theme"
+import { x } from "@xstyled/styled-components"
+import { SF } from "src/styles/styleFragments"
+import { FloatingActionButtonImPure, FloatingActionButtonImPureProps } from "."
```

Comment:
```suggestion
import { FloatingActionButtonImpure, FloatingActionButtonImpureProps } from "."
```

FloatingActionButtonImPure -> FloatingActionButtonImpure